mock/mock.go

```diff
@@ -273,6 +273,13 @@ func (c *Call) NotBefore(calls ...*Call) *Call {
 	return c
 }
 
+// InOrder defines the order in which the calls should be made
```

Comment:
This could do with a small usage example for the docs.

Reply:
It's also the order they need to be "satisfied", not the order they need to be called. In the case where they can be called multiple times.